templates/event.php

```diff
@@ -34,7 +34,10 @@
 	</div>
 	<div class="event-details-left">
 		<div class="event-page-content">
-			<?php echo esc_html( $event_description ); ?>
+			<?php
+			// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+			echo make_clickable( wp_kses_post( $event_description ) );
```

Comment:
Why not this which would not require the override?
```suggestion
			echo wp_kses_post( make_clickable( $event_description ) );
```